# Rangefinder packet data
float32 latitude
float32 longitude
float32 altitude
float32 target_distance
float32 laser_measure_info